src/js/repl.js

```diff
@@ -133,22 +135,31 @@ export function processLine(replSession: REPLSession, line: string): void {
   return undefined;
 }
 
+function stopReverseSearch(
+  replSession: REPLSession,
+  clear: boolean = true,
+): void {
+  const session = replSession;
+  session.isReverseSearch = false;
+  session.reverseSearchBuffer = '';
+
+  if (clear) {
+    session.rl.write(null, {
```

Comment:
Wouldn't `readline.clearLine` from the api also work?
 https://nodejs.org/api/readline.html#readline_readline_clearline_stream_dir